apps/emqx/src/emqx_schema.erl

```diff
@@ -288,131 +291,182 @@ fields("cache") ->
+             desc => "Maximum MQTT packet size allowed."
            })}
     , {"max_clientid_len",
        sc(range(23, 65535),
-          #{ default => 65535
+          #{ default => 65535,
+             desc => "Maximum allowed length of MQTT clientId."
            })}
     , {"max_topic_levels",
        sc(range(1, 65535),
-          #{ default => 65535
+          #{ default => 65535,
+             desc => "Maximum topic levels allowed."
            })}
     , {"max_qos_allowed",
        sc(range(0, 2),
-          #{ default => 2
+          #{ default => 2,
+             desc => "Maximum QoS allowed."
            })}
     , {"max_topic_alias",
        sc(range(0, 65535),
-          #{ default => 65535
+          #{ default => 65535,
+             desc => "Maximum Topic Alias, 0 means no topic alias supported."
           })}
     , {"retain_available",
        sc(boolean(),
-          #{ default => true
+          #{ default => true,
+             desc => "Support MQTT retained messages."
            })}
     , {"wildcard_subscription",
        sc(boolean(),
-          #{ default => true
+          #{ default => true,
+             desc => "Support MQTT Wildcard Subscriptions."
            })}
     , {"shared_subscription",
        sc(boolean(),
-          #{ default => true
+          #{ default => true,
+             desc => "Support MQTT Shared Subscriptions"
            })}
     , {"ignore_loop_deliver",
        sc(boolean(),
-          #{ default => false
+          #{ default => false,
+             desc => "Ignore loop delivery of messages for mqtt v3.1.1"
            })}
     , {"strict_mode",
        sc(boolean(),
-          #{default => false
+          #{default => false,
+            desc => "Parse the MQTT frame in strict mode"
            })
       }
     , {"response_information",
        sc(string(),
-          #{default => ""
+          #{default => "",
+            desc =>
+"""Specify the response information returned to the client
+This feature is disabled if is set to \"\"."""
            })
       }
     , {"server_keepalive",
        sc(hoconsc:union([integer(), disabled]),
-          #{ default => disabled
+          #{ default => disabled,
+             desc => "Server Keep Alive of MQTT 5.0"
            })
       }
     , {"keepalive_backoff",
        sc(float(),
-          #{default => 0.75
+          #{default => 0.75,
+            desc =>
+"""The backoff for MQTT keepalive timeout. The broker will close the connection
+after idling for 'Keepalive * backoff * 2'."""
            })
       }
     , {"max_subscriptions",
        sc(hoconsc:union([range(1, inf), infinity]),
-          #{ default => infinity
+          #{ default => infinity,
+             desc => "Maximum number of subscriptions allowed."
            })
       }
     , {"upgrade_qos",
        sc(boolean(),
-          #{ default => false
+          #{ default => false,
+             desc => "Force to upgrade QoS according to subscription."
```

Comment:
Nit: Phrasing is not ideal. Perhaps something like: Force upgrade of QoS level according to subscription.